src/ProjectDirectory.php

```diff
@@ -6,6 +6,11 @@ class ProjectDirectory
 {
     public static function find(/* string */ $cwd)
     {
+        $env = getenv('WHIPPET_PROJECT_DIRECTORY');
+        if ($env) {
+            return \Result\Result::ok(new self($env));
```

Comment:
Up to you - I genuinely don't know which is preferable - depends on use case I guess?